bootloaders/uart/uart-pt.json

```diff
@@ -0,0 +1,23 @@
+{
+  "version": [1, 0],
+  "unpartitioned": {
+    "families": ["absolute"],
+    "permissions": {
+      "secure": "rw",
+      "nonsecure": "rw",
+      "bootloader": "rw"
+    }
+  },
+  "partitions": [
+    {
+      "start": "128K",
+      "size": "32K",
+      "families": ["rp2350-arm-s"],

```

Comment:
Could this bootloader theoretically also boot the other RP2350 with a RISC-V UF2, or is this limited to Arm UF2s only?

Reply:
Yep, it all works in RISC-V too - I'll fix that